app/server/app/server/routes/api.js

```diff
@@ -148,4 +148,8 @@ module.exports = function (app) {
   });
 
   app.use('/api', router);
+
+  app.get('/api/*', (req, res) => {
+    res.status(404).json({ message: 'The api route does not exist.' });
+  });
```

Comment:
It might be more clear to use the following method:
```suggestion
  router.get('/*', (req, res) => {
    res.status(404).json({ message: 'The api route does not exist.' });
  });
  
  app.use('/api', router);
```

Reply:
Good call! This should be good to go now. I also added in 404 handling for post requests and moved the health endpoint under the api path. 

Once this gets merged, I'll have Brad update the status cake alert.